chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -104,8 +107,11 @@
           $i stop
         fi
       done
+      rm -rf /var/lib/pgsql/data
```

Comment:
could you add a comment explaining that this call will be needed for non-HA case?